### DISCUSSION TITLE
Rank Only Stats Card

### DISCUSSION THREAD

**Discussion by zgosalvez** [anuraghazra/github-readme-stats#719] [Q&A] [answered]
Is there a way to create a rank only stats card? I've tried hiding all of the other items except the rank, but the space remains, i.e. it's not _compact_. Related to #268.

**Comment by anuraghazra** [comment]
Hi @zgosalvez #721 is merged and now if you hide_rank the card size will automatically be resized to fit the content.

**Reply by nathanchu** [reply]
No, I think @zgosalvez wants the opposite, hide everything but the rank, like this:

![](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=stars,commits,prs,issues,contribs&hide_title=true)

**Reply by zgosalvez** [reply]
@nathanchu is correct

**Reply by zgosalvez** [reply]
@nathanchu, I also meant to remove the title as well. If you do that, there's large padding to the left of the rank.

**Comment by 111-365** [comment]
111

**Comment by rickstaa** [comment, marked as answer]
@zgosalvez I agree with you that the current behaviour is incorrect. I created #1649 to fix this.

**Reply by zgosalvez** [reply]
Awesome. Thanks!
